src/redux/actionTypes/experiments.js

```diff
@@ -0,0 +1,34 @@
+const EXPERIMENTS = 'experiments';
+
+/**
+ * Create experiments
+ */
+const EXPERIMENTS_CREATE = `${EXPERIMENTS}/create`;
```

Comment:
rename to `EXPERIMENTS_CREATED` and `${EXPERIMENTS}/created`;

Reply:
renamed